README.md

```diff
@@ -1,3 +1,8 @@
+[![Build Status for Linux/MacOS](https://travis-ci.org/Microsoft/napajs.svg?branch=master)](https://travis-ci.org/Microsoft/napajs)
+[![Build Status for Windows](https://ci.appveyor.com/api/projects/status/github/Microsoft/napajs?branch=master&svg=true)](https://ci.appveyor.com/project/daiyip/napajs)
```

Comment:
Both icons are showing 'build', is it possible to customize the text on the icon?